README.md

```diff
@@ -32,106 +49,31 @@ You also need to inherit your app theme from Backpack:
   </style>
 ```
 
-#### Note that Backpack is expected to be used with AndroidX. Please refer to [AndroidX migration guide](https://developer.android.com/jetpack/androidx/migrate) to setup.
+## Jetpack Compose
 
-### Demo application
-The Backpack demo application is a good way of referring to the variants available for a component and their correct usage. The code is available under `/app` directory. [The app can be downloaded from App Center](https://install.appcenter.ms/orgs/backpack/apps/backpack-android/distribution_groups/everyone) or by scanning the QR code below.
+> Backpack for Jetpack Compose status is experimental.
 
-![QR code](qr.png)
+Backpack for Compose targets **stable Compose 1.0.0** release.
 
-## Components
-
-* [Badge](docs/view/Badge/README.md)
```

Comment:
if we're no longer listing the components we should also remove the danger warning & PR template part for this